pkg/storage/storage_test.go

```diff
@@ -498,6 +499,33 @@ var _ = Describe("In-memory storage", func() {
 		`), "metrics_server_storage_points")
 		Expect(err).NotTo(HaveOccurred())
 	})
+
+	It("if node restarted, ignore calculate metrics", func() {
```

Comment:
The ginkgo framework is very expressive, so we usually want to make correct english sentences based on the matcher, in this case `It`. I would suggest something like: 
```suggestion
	It("should return nil metrics when a node restarted", func() {
```